tests/runner.d

```diff
@@ -34,7 +34,7 @@ private int main (string[] args)
     const lflags = getLflags();
     if (!importPaths.length || !lflags.length)
         return 1;
-    const Args = ["rdmd", "-vcolumns"] ~ args[1 .. $] ~
+    const Args = ["rdmd", "-vcolumns", "-i=stdx"] ~ args[1 .. $] ~
```

Comment:
vibe.d seems to pulled in stdx:allocator, let me know if you have a better idea of handling this